Unix/http/httpclient.c

```diff
@@ -540,6 +541,48 @@ static MI_Result _Sock_Write(
     return MI_RESULT_FAILED;
 }
 
+/* 
+    Writes a session id and timestamp to a trace file
+    using _WriteTraceFile.
+
+    Parameters:
+        pathId - the identifier for the trace file
+        sessionId - the string session id (HttpClient_SR_SocketData.sessionId and HttpClient.sessionId)
+
+    NOTES: This is called by _WriteClientHeader using ID_HTTPCLIENTRECVTRACEFILE
+    and _ReadHeader using ID_HTTPCLIENTSENDTRACEFILE.
+    It provides a marker to indicate the start of a request or response in the omiclient-*.trc 
+    file as well as aiding in correlating messages to a given session.
+*/
+static void _WriteTraceSessionTimestamp(_In_ PathID pathId, _In_opt_z_ const char* sessionId)
+{
+    // session id: 20, timestamp: 29, whitespace and static text:19, zero terminator
+    // [Session: NNNNN Date: YYYY/MM/DD HH:MM:SS.1234567Z]
```

Comment:
`YYYY/MM/DD` and `%04d-%02d-%02d` need to use same delimiter, thanks.

Reply:
@JumpingYang001 - I chose the format because it encodes UTC and milliseconds. While the milliseconds are not that important, the UTC is because it can be correlated to the Date header included in the response which is also UTC.

Reply:
I mean comment used `/` but the real log is dash`-`
possible better comment: `// [Session: NNNNN Date: YYYY-MM-DD HH:MM:SS.1234567Z]`